enterprise/server/backends/gcs_cache/gcs_cache.go

```diff
@@ -299,6 +300,9 @@ func (g *GCSCache) Delete(ctx context.Context, d *repb.Digest) error {
 	timer.ObserveDelete(err)
 	// Note, if we decide to retry deletions in the future, be sure to
 	// add a new metric for retry count.
+	if errors.Is(err, storage.ErrObjectNotExist) {
+		return status.NotFoundErrorf("Key %s not found in gcs_cache", d.GetHash())
```

Comment:
nit: return the full digest representation for debugging purposes, and also preserve the context from the original error:

```suggestion
		return status.NotFoundErrorf("digest %s/%d not found in gcs_cache: %s", d.GetHash(), d.GetSizeBytes(), err)
```

Reply:
Done (in new CR)